<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>librariandialog</class>
 <widget class="QDialog" name="librariandialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>798</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>管理员，欢迎使用本系统</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(158, 216, 255);</string>
  </property>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>200</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/imag/search.png);
border-image: url(:/imag/b1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>200</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/imag/search.png);
border-image: url(:/imag/b2.png);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>370</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/imag/search.png);
border-image: url(:/imag/p1.png);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>370</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/imag/search.png);
border-image: url(:/imag/p2.png);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>290</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(48, 60, 175);
font: 87 9pt &quot;Arial&quot;;
font: 14pt &quot;Arial&quot;;
font: 10pt &quot;华文新魏&quot;;
font: 11pt &quot;Arial&quot;;
font: 11pt &quot;华文新魏&quot;;</string>
   </property>
   <property name="text">
    <string>添加书籍</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>290</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(48, 60, 175);
font: 87 9pt &quot;Arial&quot;;
font: 14pt &quot;Arial&quot;;
font: 10pt &quot;华文新魏&quot;;
font: 11pt &quot;Arial&quot;;
font: 11pt &quot;华文新魏&quot;;</string>
   </property>
   <property name="text">
    <string>删除书籍</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>450</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(48, 60, 175);
font: 87 9pt &quot;Arial&quot;;
font: 14pt &quot;Arial&quot;;
font: 10pt &quot;华文新魏&quot;;
font: 11pt &quot;Arial&quot;;
font: 11pt &quot;华文新魏&quot;;</string>
   </property>
   <property name="text">
    <string>查询读者</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>450</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(48, 60, 175);
font: 87 9pt &quot;Arial&quot;;
font: 14pt &quot;Arial&quot;;
font: 10pt &quot;华文新魏&quot;;
font: 11pt &quot;Arial&quot;;
font: 11pt &quot;华文新魏&quot;;</string>
   </property>
   <property name="text">
    <string>添加读者</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>370</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/imag/search.png);
border-image: url(:/imag/p3.png);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>450</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(48, 60, 175);
font: 87 9pt &quot;Arial&quot;;
font: 14pt &quot;Arial&quot;;
font: 10pt &quot;华文新魏&quot;;
font: 11pt &quot;Arial&quot;;
font: 11pt &quot;华文新魏&quot;;</string>
   </property>
   <property name="text">
    <string>删除读者</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>530</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/imag/search.png);
border-image: url(:/imag/code.png);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>530</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/imag/search.png);
border-image: url(:/imag/pay.png);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>620</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(48, 60, 175);
font: 87 9pt &quot;Arial&quot;;
font: 14pt &quot;Arial&quot;;
font: 10pt &quot;华文新魏&quot;;
font: 11pt &quot;Arial&quot;;
font: 11pt &quot;华文新魏&quot;;</string>
   </property>
   <property name="text">
    <string>缴纳罚款</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>620</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(48, 60, 175);
font: 87 9pt &quot;Arial&quot;;
font: 14pt &quot;Arial&quot;;
font: 10pt &quot;华文新魏&quot;;
font: 11pt &quot;Arial&quot;;
font: 11pt &quot;华文新魏&quot;;</string>
   </property>
   <property name="text">
    <string>修改密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>391</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/imag/1.png);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>290</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(48, 60, 175);
font: 87 9pt &quot;Arial&quot;;
font: 14pt &quot;Arial&quot;;
font: 10pt &quot;华文新魏&quot;;
font: 11pt &quot;Arial&quot;;
font: 11pt &quot;华文新魏&quot;;</string>
   </property>
   <property name="text">
    <string>书籍信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/imag/search.png);
border-image: url(:/imag/b3.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
